demos/boussinesq/boussinesq.py.rst

```diff
@@ -0,0 +1,269 @@
+Steady Boussinesq problem with integral constraints
```

Comment:
Please don't also attempt to re-merge the Boussinesq demo.

Reply:
I have absolutely no idea how this got here, or why I now have so many changed files after merging.